paddle/go/recordio/multi_reader.go

```diff
@@ -0,0 +1,140 @@
+package recordio
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+// MultiScanner is a scanner for multiple recordio files.
+type MultiScanner struct {
+	paths      []string
+	curFile    *os.File
+	curScanner *Scanner
+	pathIdx    int
+	end        bool
+	err        error
+}
+
+// NewMultiScanner creates a new MultiScanner.
+func NewMultiScanner(paths []string) (*MultiScanner, error) {
```

Comment:
`[]string` to `...string`, so that it can accept only one filename pattern easily?